src/openrct2/ride/Vehicle.cpp

```diff
@@ -878,11 +878,11 @@ static uint8_t vehicle_sounds_update_get_pan_volume(OpenRCT2::Audio::VehicleSoun
     if (pan_x > 0)
     {
         pan_x = (0x400 - pan_x) / 4;
-        vol2 = LOBYTE(pan_x);
-        if (static_cast<int8_t>(HIBYTE(pan_x)) != 0)
+        vol2 = LoByte(pan_x);
+        if (static_cast<int8_t>(HiByte(pan_x)) != 0)
         {
             vol2 = 0xFF;
-            if (static_cast<int8_t>(HIBYTE(pan_x)) < 0)
+            if (static_cast<int8_t>(HiByte(pan_x)) < 0)
```

Comment:
This code just seems a bit odd

Reply:
What would you prefer instead?

Reply:
As discussed on discord for follow on investigating